### PR TITLE
Flowtypes - what do you guys think?

### PR DESCRIPTION
**Pull request by kalmanb**
I thought it came out quite nice

### CODE REVIEW

**Comment by kalmanb** [comment]
Note: only done for JsonDoc - not the whole app. I also changes `spec` to `service` to align with apidoc terminology.

**Comment by kalmanb** [comment]
To test - `npm run flow`

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["movio"],
-  "plugins": []
+  "plugins": ["transform-flow-strip-types"]
```
Comment (d4ncer) [inline .babelrc]: Don't you need `"syntax-flow"` as well, for this to work? https://babeljs.io/docs/plugins/syntax-flow/

Comment (kalmanb) [inline .babelrc]: I wonder if it's an either or? I have no idea

**Pull request merged**